webcompat/templates/home-page/form.html

```diff
@@ -57,7 +57,7 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
             </div>
             <span class="wc-Form-validation">
             {{ form.description(class_='wc-Form-item required',
-               placeholder='e.g. The video should have started after 3 sec.',
+               placeholder="e.g. The video doesn't play",
```

Comment:
Is there a reason for using `"` and not ` ' `? Just curious. :)

Otherwise it looks good.